src/SFA.DAS.AssessorService.Application.Api/Controllers/Staff/StaffReportsController.cs

```diff
@@ -115,7 +115,7 @@ public async Task<IActionResult> GetReport(Guid reportId)
             }
             catch (SqlException sqlEx)
             {
-                _logger.LogInformation($"Could not generate report due to : {sqlEx.Message}");
+                _logger.LogError($"Could not generate data from report Id [{reportId}] due to : {sqlEx.Message}");
                 return NoContent();
```

Comment:
Here we're logging an error, but returning a success status. That doesn't seem consistent
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204

Reply:
@ChrisJWoodcock  please see above